Libraries/LibGUI/GItemView.h

```diff
@@ -48,6 +48,8 @@ class GItemView : public GAbstractView {
     int m_visual_column_count { 0 };
     int m_visual_row_count { 0 };
 
+    bool mightdrag { false };
```

Comment:
Member variables use the `m_variable_name` convention (the `m_` is for "member" :)
Also, since "might drag" is two words, please call it `m_might_drag`